builtin/credential/ldap/path_config.go

```diff
@@ -348,55 +349,62 @@ func (c *ConfigEntry) GetTLSConfig(host string) (*tls.Config, error) {
 }
 
 func (c *ConfigEntry) DialLDAP() (*ldap.Conn, error) {
-
-	u, err := url.Parse(c.Url)
-	if err != nil {
-		return nil, err
-	}
-	host, port, err := net.SplitHostPort(u.Host)
-	if err != nil {
-		host = u.Host
-	}
-
+	var retErr *multierror.Error
 	var conn *ldap.Conn
-	var tlsConfig *tls.Config
-	switch u.Scheme {
-	case "ldap":
-		if port == "" {
-			port = "389"
-		}
-		conn, err = ldap.Dial("tcp", host+":"+port)
+	urls := strings.Split(c.Url, ",")
+	for _, uut := range urls {
+		u, err := url.Parse(uut)
 		if err != nil {
-			break
+			retErr = multierror.Append(retErr, fmt.Errorf("error parsing url %q: %s", uut, err.Error()))
```

Comment:
I did realize this but it felt to me like as long as something works it's good enough. Maybe we could return a second error that if populated goes into the response warning, but that doesn't feel like something that you should be showing the client. Maybe put it into Vault's log?